### PR TITLE
Add shell criteria token

### LINKED ISSUE #2044: Add [shell=xdg_shell] style criteria

### PR DESCRIPTION
**Pull request by RyanDwyer**
Closes #2044.

### CODE REVIEW

**Comment by RyanDwyer** [comment]
@SirCmpwn Is that build failure my problem? I can't see what I'm doing wrong.

    [2/212] Generating sway.5 with a custom command.
    FAILED: sway.5 
    /usr/bin/sh -c '/usr/bin/scdoc < ../sway/sway.5.scd > sway.5'
    Error at 559:1: Cannot nest inline formatting.

**Comment by RedSoxFan** [comment]
On the first line of the description, you have `xdg_shell`. The underscore needs to be escaped to make it `xdg\_shell`.

File: sway/criteria.c [hunk 1]
```diff
@@ -271,6 +283,9 @@ static char *get_focused_prop(enum criteria_token token) {
 	case T_INSTANCE:
 		value = view_get_instance(view);
 		break;
+	case T_SHELL:
+		value = view_get_type(view);
```
Comment (emersion) [inline sway/criteria.c]: Maybe `view_get_type` should be renamed `view_get_shell` for consistency?

**Comment by emersion** [comment]
Thanks!

**Pull request merged**